ftw/simplelayout/locales/ftw.simplelayout.pot

```diff
@@ -402,6 +402,16 @@ msgstr ""
 msgid "description_teaser_fieldset"
 msgstr ""
 
+#. Default: "The selected Content cannot be selected"
```

Comment:
I'd lowercase `Content` for the English :-)

Reply:
I'll change that